lib/mailgun/events/events.rb

```diff
@@ -87,12 +87,22 @@ def get_events(params = nil, paging = nil)
     #
     # Return is irrelevant.
     def extract_paging(response)
-      # This is pretty hackish. But the URL will never change in API v2.
-      @paging_next = response.to_h['paging']['next'].split('/')[6]
-      @paging_previous = response.to_h['paging']['previous'].split('/')[6]
-    rescue
-      @paging_next = nil
-      @paging_previous = nil
+      next_page_url = response.to_h.dig('paging', 'next')
```

Comment:
I like this approach but we need to be backwards compatible with at least Ruby 2.0.0. If you could also include a gem like [this](https://github.com/Invoca/ruby_dig) to provide backwards-compat, I can for sure merge this in. Thanks!